admin_frontend/src/components/ApiKeys.jsx

```diff
@@ -61,13 +68,20 @@ const ApiKeysPage = () => {
 			toast.error("Please input both API_KEY & PRIVATE_KEY field");
 			return;
 		}
+		if (!/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*-_&])[A-Za-z\d@$!%*-_&]{8,}$/.test(apiKey)) {
+			toast.error("Invalid Validation: API_KEY format. Please see the docs for more info");
+			return;
+		}
 		try {
 			setLoading(true);
 			const requestData = {
 				API_KEY: apiKey,
 				PRIVATE_KEY: privateKey,
 				SUPPORTED_NETWORKS: supportedNetworks ?? "",
 				ERC20_PAYMASTERS: customErc20Paymaster ?? "",
+				TRANSACTION_LIMIT: txnMode,
+        NO_OF_TRANSACTIONS_IN_A_MONTH: noOfTxn,
+        INDEXER_ENDPOINT: 'http://localhost:3003'
```

Comment:
Is this not something that should be configurable?

Reply:
added in .env file